package.json

```diff
@@ -4,7 +4,7 @@
   "description": "GraphicaL User Interface for creating and running Scratch 3.0 projects",
   "main": "./src/index.js",
   "scripts": {
-    "build": "npm run clean && webpack --progress --colors --bail",
```

Comment:
FWIW, we do have it set up that way on www https://github.com/LLK/scratch-www/blob/develop/Makefile#L11. I would also rather have build do the clean step, since it ensures you don't have stray/stale files, which I think is important with all of the `npm link`ing we do, and switching branches when testing PRs etc.

I'll also note that `clean` also uses the Windows-unfriendly `mkdir -p`. But the `rimraf` README points out that can be resolved with another package, `mkdirp`😛